redev_assert.h

```diff
@@ -4,7 +4,7 @@
 #define REDEV_ALWAYS_ASSERT(cond) do {          \
   if (! (cond)) {                               \
     char omsg[2048];                            \
-    sprintf(omsg, "%s failed at %s + %d \n",    \
+    snprintf(omsg,200, "%s failed at %s + %d \n",    \
```

Comment:
Will 200 chars be enough for all the conditions + function names?

Reply:
Good catch! This should be the buffer size which is 2048.